src/main/groovy/com/epam/petclinic/plugin/DatabasePlugin.groovy

```diff
@@ -0,0 +1,119 @@
+package com.epam.petclinic.plugin
+
+import groovy.sql.Sql
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+/**
+ * Database plugin for liquibase tasks, creation and deletion database.
+ *
+ * Date: 5/16/2017
+ *
+ * @author Stanislau Halauniou
+ */
+class DatabasePlugin implements Plugin<Project> {
+
+    private static final String LIQUIBASE_PLUGIN = 'liquibase'
+    private static final String POSTGRESQL = 'postgresql'
+    private static final String DATABASE_TASK_GROUP = 'Database Applier'
+    private static final String POSTGRES_DRIVER = 'org.postgresql.Driver'
+    private static final String JDBC_URL = 'jdbc:postgresql://localhost:5432/'
+    private static final String CREATE_DATABASE = 'createDatabase'
+    private static final String DROP_DATABASE = 'dropDatabase'
+
+    @Override
+    void apply(Project project) {
+        project.plugins.apply(LIQUIBASE_PLUGIN)
+
+        initJdbcDriver(project)
+        configureLiquibase(project)
+        addDatabaseTasks(project)
+    }
+
+    /**
+     * Initiates JDBC driver.
+     *
+     * @param project project to which plugin applied
+     */
+    private void initJdbcDriver(Project project) {
+        // Finds a jdbc driver
+        File driver = project.rootProject.buildscript.configurations.classpath.find { File file ->
+            file.name.contains(POSTGRESQL)
+        }
+        // Loads and registers a JDBC driver for database access
+        URLClassLoader loader = GroovyObject.class.classLoader
+        loader.addURL(driver.toURI().toURL())
```

Comment:
Could you explain how it will work? I just want to be sure we load right things in any case... For example what will happen if we have two libraries which contains `postgresql` in their names?